config/routes.rb

```diff
@@ -1,5 +1,9 @@
 Rails.application.routes.draw do
-  devise_for :users
+  devise_for :users, skip: [:registrations]
```

Comment:
Is this the preferred way to do this? It feels weird to skip registrations and then define manual paths below.